web/src/app/services/IntegrationKeyDeleteDialog.tsx

```diff
@@ -45,7 +50,7 @@ export default function IntegrationKeyDeleteDialog(props) {
   const [deleteKey, deleteKeyStatus] = useMutation(mutation, {
     onCompleted: props.onClose,
     update: (cache) => {
-      const { service } = cache.readQuery({
+      const { ...service } = cache.readQuery({
```

Comment:
```suggestion
      const { service } = cache.readQuery({
```

We'll want to continue explicitly calling for the service prop, as destructuring it here under the name `service` will grab everything from the result of `readQuery`, which we don't want (we only want the `service` object from the cache.

With the destructuring the following warning/errors show in the console after deleting:
![image](https://user-images.githubusercontent.com/11381794/86816801-e37d3280-c049-11ea-8264-408e6f6d5db4.png)